playground/src/app.js

```diff
@@ -80,6 +81,10 @@ events.subscribe(ErrorList.EVENT_ERROR_SELECTED, (error) =>
   editor.setCursorAndFocus(error.line, error.col)
 );
 
+// configure background layer to close fly ins
+const flyInBackground = document.getElementById('fly-in-background');
+FlyInBackground.createFlyInBackground(flyInBackground);
```

Comment:
I started to refactor statements like this to the following:

https://github.com/ampproject/amp.dev/blob/3871b0b59494633d2ef7b2b3e4ff796cbea7e114/playground/src/app.js#L69

https://github.com/ampproject/amp.dev/blob/3871b0b59494633d2ef7b2b3e4ff796cbea7e114/playground/src/error-list/error-list.js#L26-L35

This cleans up app.js while at the same time allowing us to do specific case switches. Also almost all of our components are singletons so it doesn't make much sense to keep the factory functions configurable.